lib/systemd_testsuite_test.pm

```diff
@@ -19,33 +18,11 @@ use warnings;
 use known_bugs;
 use testapi;
 use power_action_utils 'power_action';
+use repo_tools 'add_qa_head_repo';
```

Comment:
Invalid for systemd-testsuite

Reply:
@SergioAtSUSE ok, I'll remove change for lib/systemd_testsuite_test.pm. Thx for review!